bin/deploy-wmagent.sh

```diff
@@ -268,6 +270,7 @@ echo "Done!" && echo
 echo "*** Tweaking configuration ***"
 sed -i "s+team1,team2,cmsdataops+$TEAMNAME+" $MANAGE/config.py
 sed -i "s+Agent.agentNumber = 0+Agent.agentNumber = $AG_NUM+" $MANAGE/config.py
+sed -i "s+BossAir.pluginNames = \['PyCondorPlugin+BossAir.pluginNames = \['SimpleCondorPlugin+" $MANAGE/config.py
```

Comment:
I'm going to update it directly in the default values, so it won't be needed at this stage.